sonar-api-model/build.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+    id("java-library")
+    id("maven-publish")
+}
+
```

```suggestion

description = "Library with API models for data from Sonar tools"
```

Maven Central requires a description for every module that's published